src/docs/sphinx/quickstart_guide/config_build.rst

```diff
@@ -120,38 +145,40 @@ Building and Installing Third-party Libraries
 ---------------------------------------------
 
 We use the `Spack Package Manager <https://github.com/spack/spack>`_
-to manage and build TPL dependencies for Axom. The Spack process works on Linux and macOS
-systems. Axom does not currently have a tool to automatically build dependencies for
-Windows systems.
+to manage and build TPL dependencies for Axom. The Spack process works on 
+Linux and macOS systems. Although we test Axom on Windows and some Axom users
+build their applications for Windows, Axom does not currently have a tool to 
+automatically build dependencies for Windows systems.
```

Comment:
This is out of date w.r.t. windows.
Something along the following line might be better:

> We use the `Spack Package Manager <https://github.com/spack/spack>`_
to manage and build TPL dependencies for Axom on Linux and macOS systems.
Similarly, support for managing and building TPLs on Windows is provided through `Vcpkg <https://github.com/microsoft/vcpkg>`_,

Reply:
fixed.